util-core/src/main/scala/com/twitter/util/Config.scala

```diff
@@ -134,43 +135,65 @@ trait Config[T] extends (() => T) {
     val buf = new mutable.ListBuffer[String]
     val mirror = scala.reflect.runtime.universe.runtimeMirror(getClass.getClassLoader)
     def collect(prefix: String, config: Config[_]): Unit = {
+      CoverageChecker.initialize("collect", 14)
       if (!alreadyVisited.contains(config)) {
+        CoverageChecker.reached("collect", 0)
         alreadyVisited += config
         val nullaryMethods = config.getClass.getMethods.toSeq filter { _.getParameterTypes.isEmpty }
         val syntheticTermNames: Set[String] = {
           val configSymbol = mirror.reflect(config).symbol
           val configMembers = configSymbol.toType.members
           configMembers.iterator.collect {
-            case symbol if symbol.isTerm && symbol.isSynthetic =>
```

Comment:
I might be wrong here, but shouldn't there be a branch for the default case, i.e. when `if symbol.isTerm && symbol.isSynthetic` doesn't hold? What i mean is to add:
> ` case _ : => CoverageChecker.reached("collect", 2)`

Reply:
See line 178

Reply:
This causes a syntax error due to collect being used, so we can't actually test that :/